File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 1) return 1;
        int cont = 1;
        vector<int> new_nums = {nums[0]};

        for(int i = 0, j = 0; i<nums.size(); i++){
            if(new_nums[j] != nums[i]){
                cont++;
                j++;
                new_nums.push_back(nums[i]);
            }
        }
        nums = new_nums;
        return cont;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if(len(nums) == 1):
            return 1

        cont = 1
        check = nums[0]
        new_nums = []
        new_nums.append(check)

        for number in nums:
            if number != check:
                cont += 1
                new_nums.append(number)
                check = number

        nums[:] = new_nums  # Update the input list in-place
        
        return cont        

        